linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_ownership_binary_dirs/rule.yml

```diff
@@ -28,11 +28,11 @@ identifiers:
     cce@rhel7: CCE-82048-0
     cce@rhel8: CCE-80806-3
     cce@rhel9: CCE-83908-4
+    cce@sle12: CCE-83241-0
     cce@sle15: CCE-85730-0
 
 references:
-    nist: CM-6(a),AC-6(1)
-    nist@sle15: CM-5(6),CM-5(6).1
+    nist: CM-5(6),CM-5(6).1,CM-6(a),AC-6(1)
     cci@sle15: CCI-001499
```

Comment:
Please remove this line. CCIs are marked as "disa:" and they are platform independent, so this rule already has it correctly.

Reply:
thanks for noticing :bow: - now 46e9a5b  should be ok.
Also rebased the whole PR